doc/classes/Node.xml

```diff
@@ -130,6 +130,13 @@
 				[b]Note:[/b] This method is only called if the node is present in the scene tree (i.e. if it's not an orphan).
 			</description>
 		</method>
+		<method name="accept_event">
+			<return type="void" />
+			<description>
+				Stops the input from propagating further down the [SceneTree].
```

Comment:
Assuming it still is accurate, the new description is a bit too vague... or wrong? I noticed it's identical to Viewport's, but... hm. For example, I remember that the actual order is dependent on the current input step, it's not always "down".
```suggestion
				Stops the input event from propagating further in this node's viewport (see [method get_viewport]). This is equivalent to [method Viewport.set_input_as_handled] on this node's viewport.
```

Reply:
Emm... I copy it from the Viewport::set_input_as_handled, since its behavior is the same as it. Is this means that the description in Viewport also need change?

Reply:
Doesn't have to be done in this PR